app/views/users/index.html.erb

```diff
@@ -1,5 +1,21 @@
-<div class="users">
-<% @users.each do |user| %>
-  <%= render partial: 'user', locals: { user: user } %>
-<% end %>
-</div>
\ No newline at end of file
+<section class='main_section'>
+  <h1>Here is a list of all users</h1>
+  <ul class='users_list'>
+    <% @users.each do |user| %>
+      <li class='user_card'>
+        <div class='user_img'>
+          <img src= <%= user.photo %> alt= <%= user.name %> />
+        </div>
+
+        <div class='user_stuff'>
+          <p class='user_name'><%= link_to user.name, user_path(user) %></p>
+          <p class='posts_no'>Number of Posts: <%= user.posts_counter || 0 %></p>
+        </div>
+      </li>
+    <% end %>
+</ul>
+
+<div>
+<a href="<%= url_for("/users/#{@user.id}/posts/new") %>">Create new post</a>
```

Comment:
- [ ] It will be good to ensure that there are no errors in your application. When I run the project, the error below pops up ⬇️ 
![Screenshot 2023-05-27 at 14 32 54](https://github.com/LeslieAine/blog-app1/assets/32361963/1ed15b95-e7c4-41ca-a135-e64fd31945ff).

I tried fixing this error and it seems you are calling the `@user.id ` outside of the loop. When I resolved this issue another issue popped up in `user/show `page